angler/config/system-proprietary-blobs-api24.txt

```diff
@@ -1,5 +1,6 @@
 system/bin/fingerprintd
 system/bin/mm-qcamera-app
+system/bin/nanoapp_cmd
```

Comment:
The AOSP build indeed differs from factory image. Do you have any idea why? Have you spotted any issues when using the AOSP version?
```
$ diff out_release__7.0.0_r6/aosp_prod/target/product/angler/system/bin/nanoapp_cmd ../nexus-vendor-blobs/angler/nrd90u/factory_imgs_data/system/bin/nanoapp_cmd 
Binary files out_release__7.0.0_r6/aosp_prod/target/product/angler/system/bin/nanoapp_cmd and ../nexus-vendor-blobs/angler/nrd90u/factory_imgs_data/system/bin/nanoapp_cmd differ
$ md5sum out_release__7.0.0_r6/aosp_prod/target/product/angler/system/bin/nanoapp_cmd ../nexus-vendor-blobs/angler/nrd90u/factory_imgs_data/system/bin/nanoapp_cmd 
cbb83b93bb7d56d27213694579d49b9c  out_release__7.0.0_r6/aosp_prod/target/product/angler/system/bin/nanoapp_cmd
78fc8467f727eea5481cfd02b6e81ba5  ../nexus-vendor-blobs/angler/nrd90u/factory_imgs_data/system/bin/nanoapp_cmd
```